kod/object/active/holder/room.kod

```diff
@@ -886,11 +886,6 @@ messages:
          return RID_KOCATAN;
       }
 
-      if (piRoom_num = RID_GUILDH12) or (piRoom_num = RID_GUILDH13)
```

Comment:
Did you forget these?

Reply:
These two rooms already override `GetRegion` and return `RID_KOCATAN`.